src/General/Storage.Net.Ftp/Storage.Net.Ftp.csproj

```diff
@@ -5,11 +5,14 @@
       <LangVersion>latest</LangVersion>
       <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
       <WarningsAsErrors />
+      <Version>1.1.0</Version>
+      <FileVersion>1.1.0.0</FileVersion>
+      <AssemblyVersion>1.1.0.0</AssemblyVersion>
```

Comment:
Version seems to be not required as the solution gets a version assigned on release.